news.Rmd

```diff
@@ -95,7 +95,7 @@ Functions, arguments, and file names should be wrapped in backticks.
 * In `stat_bin()`, `binwidth` now also takes functions.
 
 # Bad
-* In the `stat_bin` function, the `binwidth` argument now also takes functions.
+* In the stat_bin function, the "binwidth" argument now also takes functions.
```

Comment:
Maybe add "the ... argument" in the next section to emphasise that if you use this style you don't need to describe whether something is an argument or function?

Reply:
like https://github.com/tidyverse/style/pull/135/commits/ad7126b7a98d0d8297688fea3711919faaec3284 you mean?